base/string_utils.cpp

```diff
@@ -326,4 +326,29 @@ bool AlmostEqual(string const & str1, string const & str2, size_t mismatchedCoun
   return false;
 }
 
+void Split(string const & s, char delimiter, vector<string> & target)
```

Comment:
I don't see how much this method faster. But it's clear that it mostly duplicates what SimpleTokenizer does.